runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/status/DebugCapture.java

```diff
@@ -43,6 +44,8 @@
  * DebugCapture encapsulates a simple periodic sender for HTML pages to the debug capture service.
  * It is dynamically configured by the service.
  */
+// The fields are written but only implicitly read. Not a bug.
+@SuppressFBWarnings("URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD")
```

Comment:
Can this supporession be added to the fields themselves? It's unclear what this is applied to.

Reply:
Done